Add more-info for System Health (#16546)
home-assistant/home-assistant.github.io: source/more-info/system-health.markdown

---
title: "System Health"
description: "More information about System Health."
---

System health is the card on the Information part of the configuration section in the Home Assistant UI.

> This provide information about your environment and enabled integrations.

Reviewer: ```suggestion
This provides information about your environment and enabled integrations.
```

To find this card first navigate to the Configuration panel, then scroll all

> the way to the bottom and select "Info". At the bottom of that page you will see the  System Health card.

Reviewer: ```suggestion
the way to the bottom and select "Info". At the bottom of that page, you will see the  System Health card.
```

![sytem_health](/images/more-info/system_health.png)

## GitHub issues

> GitHub issues often calls for the date presented in the System Health.

Reviewer: ```suggestion
GitHub issues often call for the data presented in the System Health.
```
